llvm/lib/Analysis/InlineCost.cpp

```diff
@@ -3272,6 +3276,9 @@ InlineCost llvm::getInlineCost(
     return llvm::InlineCost::getNever(UserDecision->getFailureReason());
   }
 
+  if (InlineAllViableCalls && isInlineViable(*Callee).isSuccess())
+    return llvm::InlineCost::getAlways("inline all viable calls");
```

Comment:
Nit: I'd make it more specific. E.g.  "Inlining forced by --inline-all-viable-calls"